test/python/transpiler/test_unroller.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Test the Unroller pass"""
+
+from sympy import pi
+
+from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
+from qiskit.transpiler.passes import Unroller
+from qiskit.converters import circuit_to_dag
+from ..common import QiskitTestCase
+
+
+class TestUnroller(QiskitTestCase):
+    """Tests the Unroller pass."""
+
+    def test_basic_unroll(self):
+        """Test decompose a single H into u2.
+        """
+        qr = QuantumRegister(1, 'qr')
+        circuit = QuantumCircuit(qr)
+        circuit.h(qr[0])
+        dag = circuit_to_dag(circuit)
+        pass_ = Unroller(['u2'])
+        unrolled_dag = pass_.run(dag)
+        op_nodes = unrolled_dag.get_op_nodes()
+        self.assertEqual(len(op_nodes), 1)
+        op = unrolled_dag.multi_graph.nodes[op_nodes.pop()]["op"]
```

Comment:
I would avoid accessing `multi_graph` from "outside", since is not part of the DAG API.

Reply:
ok, done.